packages/marketplace/contracts/transfer-manager/TransferExecutor.sol

```diff
@@ -28,7 +28,7 @@ abstract contract TransferExecutor is Initializable, ITransferExecutor {
             //not using transfer proxy when transferring from this contract
             address token = abi.decode(asset.assetType.data, (address));
             SafeERC20Upgradeable.safeTransferFrom(IERC20Upgradeable(token), from, to, asset.value);
-        } else if (asset.assetType.assetClass == LibAsset.AssetClassType.ERC1155_ASSET_CLASS) {
+        } else {
```

Comment:
it reverting if asset.assetType.assetClass is not part of AssetClassType ?

Reply:
We have INVALID_ASSET_CLASS, even if it doesn't happen here it may be a good idea to revert.